Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.xaml

```diff
@@ -20,20 +20,14 @@
                             <VisualStateGroup x:Name="GridSplitterStates">
                                 <VisualState x:Name="Normal" />
                                 <VisualState x:Name="PointerOver">
-                                    <Storyboard>
-                                        <ColorAnimation Duration="0"
-                                                        To="{ThemeResource SystemBaseLowColor}"
-                                                        Storyboard.TargetProperty="(Grid.Background).(SolidColorBrush.Color)"
-                                                        Storyboard.TargetName="RootGrid" />
-                                    </Storyboard>
+                                    <VisualState.Setters>
+                                        <Setter Target="RootGrid.Background" Value="{ThemeResource SystemControlBackgroundBaseLowBrush}" />
+                                    </VisualState.Setters>
                                 </VisualState>
                                 <VisualState x:Name="Pressed">
-                                    <Storyboard>
-                                        <ColorAnimation Duration="0"
-                                                        To="{ThemeResource SystemBaseHighColor}"
-                                                        Storyboard.TargetProperty="(Grid.Background).(SolidColorBrush.Color)"
-                                                        Storyboard.TargetName="RootGrid" />
-                                    </Storyboard>
+                                    <VisualState.Setters>
+                                        <Setter Target="RootGrid.Background" Value="{ThemeResource SystemControlBackgroundBaseHighBrush}" />
```

Comment:
Should probably keep the colors the same as they were

Reply:
@skendrot the brushes that I've used are SolidColorBrushes with the same colors. Their definitions in generic.xaml are,

```XAML
<SolidColorBrush x:Key="SystemControlBackgroundBaseHighBrush" Color="{StaticResource SystemBaseHighColor}"/>
<SolidColorBrush x:Key="SystemControlBackgroundBaseLowBrush" Color="{StaticResource SystemBaseLowColor}"/>
```

Reply:
You should probably use ThemeResource for those two color definitions rather than StaticResource.

Reply:
@JohnnyWestlake those brushes are defined in the generic.xaml file that ships with the SDK, located at `C:\Program Files (x86)\Windows Kits\10\DesignTime\CommonConfiguration\Neutral\UAP\10.0.16299.0\Generic` in my case.

Reply:
@skendrot @nmetulev ?

Reply:
Thanks for the clarification on the colors @RavinduL! I would agree with @JohnnyWestlake that they should be theme resource defined so if the theme of the application changes they will update also

Reply:
@skendrot the Brushes that use StaticResources are from the generic.xaml file that ships with the Windows SDK as I've mentioned here: https://github.com/Microsoft/WindowsCommunityToolkit/pull/2263#discussion_r197162027

Reply:
Yes, but you can still use them as a theme resource

Reply:
@skendrot and that is what I've done on lines 24 and 29 of the `GridSplitter.xaml` file. Are you referring to something else? @JohnnyWestlake